src/main/Main.ts

```diff
@@ -1,12 +1,52 @@
-import {app, ipcMain, nativeImage, BrowserWindow, screen} from "electron";
+import {app, nativeImage, BrowserWindow, screen, globalShortcut, Menu} from "electron";
 import {readFileSync} from "fs";
 import {windowBoundsFilePath} from "../utils/Common";
+import {buildMenuTemplate} from "./Menu";
+
+/**
+ * Contains references to all currently open windows.
+ *
+ * @type {Array}
+ */
+let windows: Electron.BrowserWindow[] = [];
```

Comment:
So what should i do? I'd like to avoid the initial suggestion from @drew-gross (sorry) because it's an additional shifting around of data, especially given that it's gonna be replaced by a zipper. In order to achieve a similar outcome for the meantime, we could just introduce a `getFocussedWindow()` function for ease of access, what do you think?